client/src/components/ResultData.js

```diff
@@ -1,10 +1,19 @@
 export default class ResultData {
-  constructor(title, description, excerpt, location, postUrl, applyUrl) {
+  constructor(title, link, description, excerpt, city, state, country, latitude, longitude, companyName, minSalary, startDate) {
```

Comment:
excerpt is not included in the database. We can generate this by shortening the description and appending an ellipsis to the end.

Reply:
Great!

Reply:
By the way, title should be job_title based on the most up-to-date version of the schema.